weave/wandb_interface/wandb_artifact_pusher.py

```diff
@@ -46,17 +49,25 @@ def write_artifact_to_wandb(
 
     if _lite_run is None:
         lite_run = InMemoryLazyLiteRun(
-            entity_name, project_name, group="weave_artifact_pushers", _hide_in_wb=True
+            entity_name,
+            project_name,
+            group="weave_artifact_pushers",
+            _hide_in_wb=True,
+            _always_create_project=_create_new_artifact,
+            _fetch_last_history_step=_create_new_artifact,
```

Comment:
Why do you want to fetch the last history step when you are creating a new artifact? I would imagine this should always be False here

Reply:
Creating a brand new artifact (collection) is the default path and for that, we need to make sure that a project is upserted. But fetching the last history step is not necessary here.  Agreed, I'll change it to False!